src/charselect.cpp

```diff
@@ -365,7 +365,7 @@ void Courtroom::filter_character_list()
       continue;
     }
 
-    if (!char_list.at(i).name.contains(ui_char_search->text(), Qt::CaseInsensitive))
+    if (!char_list.at(i).name.contains(ui_char_search->text(), Qt::CaseInsensitive) && !ao_app->get_category(char_list.at(i).name).contains(ui_char_search->text(), Qt::CaseInsensitive)) {
```

Comment:
Your code does not seem to compile due to an extra {.
Also, make sure to make changes based on master and update as needed before PRing.

I enabled CI runs for you so you know if your formatting is good too. You can enable auto-formatting in your IDE usually.

Reply:
What a bad mistake, sorry.
I'll fix as soon as I can